topics/academic-integrity.qmd

```diff
@@ -0,0 +1,45 @@
+---
+title: Academic Integrity
+---
+
+## Netherlands Code of Conduct for Research Integrity
+
+Dutch scientists are required to comply with the [Netherlands Code of Conduct for Research Integrity](https://www.universiteitenvannederland.nl/en/research-integrity) (VSNU, 2018). The principles of proper scientific and scholarly research, according to the Code of Conduct are:
+
+* Honesty
+* Scrupulousness
+* Transparency
+* Independence
+* Responsibility
+
+The principles of honesty and transparency state explicit guidelines for the way in which you treat your research data:
+
+* Honesty: you should refrain from fabricating or falsifying data
+* Transparency:
+  * You should ensure that it is clear to others what data your research is based on, how the data were obtained, what the results are and how you got to these results
+  * All steps in your research process must be verifiable (e.g. choice of research question, research design, methodology, sources used), so that it is clear to others how your research was conducted
+
+To live up to these general principles, the Code of Conduct provides the following standards, which are addressed in a DMP, for good research practices related to data management:
+
+* Provide a description of the way in which the collected research data are organised and classified, so that they can be verified and re-used (standard 3.2.10)
+* Make research data public upon completion of your research project; if this is not possible, explain why (standards 3.2.11 and 3.4.45)
+* Describe the data you have collected and used in your research honestly, scrupulously and transparently (standard 3.3.23)
+* Manage your data carefully and store both the raw and processed versions for a period appropriate for your discipline (standard 3.3.24)
+* Contribute towards making data [FAIR](./fair-principles.qmd), where possible (standard 3.3.25)
+* Be transparent about your methods and working procedures by using e.g. research protocols, logs, lab journals or reports to describe these processes (standard 3.4.35)
+
+## Academic integrity at VU Amsterdam
+
+To protect academic integrity VU Amsterdam and Amsterdam UMC, location VUmc subscribe to the [Netherlands Code of Conduct for Research Integrity](https://www.universiteitenvannederland.nl/en/research-integrity). On the [Academic Integrity page](https://vu.nl/en/about-vu/more-about/academic-integrity) on the VU website, you can find more information about how these organisations implement the duties of care for institutions to uphold the principles of academic integrity.
```

Comment:
```suggestion
To protect academic integrity at VU Amsterdam and Amsterdam UMC, location VUmc subscribes to the [Netherlands Code of Conduct for Research Integrity](https://www.universiteitenvannederland.nl/en/research-integrity). On the [Academic Integrity page](https://vu.nl/en/about-vu/more-about/academic-integrity) on the VU website, you can find more information about how these organisations implement the duties of care for institutions to uphold the principles of academic integrity.
```

The first sentence does not flow well at the moment. Might there be a way to take another look at this? I am not sure I understand the "location VUmc" reference.